api/lib/app_messaging_service.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module AppMessagingService # rubocop:disable Style/Documentation
```

Comment:
Do not disable Rubocop linter. Remove this comment.
Try adjusting your code according to lint description instead.